addons/interface/functions/script_component.hpp

```diff
@@ -1 +1 @@
-#include "../script_component.hpp"
+#include "\dynulo\radio\addons\interface\script_component.hpp"
```

Comment:
Is there a reason behind this change?

Reply:
Convention mostly from ACE-like projects. The `manager` component also has it like this.